### PR TITLE
Add onetracker docs

### PR DESCRIPTION
**Pull request by packruler**
## Proposed change

Add docs for OneTracker integration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/86921
- Link to parent pull request in the Brands repository:  https://github.com/home-assistant/brands/pull/4088

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/onetracker.markdown [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  - Postal Service
+ha_release: 2023.03
+ha_iot_class: Cloud Polling
+ha_domain: OneTracker
+ha_platforms:
+  - sensor
+ha_integration_type: integration
+---
+
+The `OneTracker` platform allows one to track deliveries by [OneTracker](https://onetracker.app), a service that supports 490+ couriers worldwide. To use the tracking API functionality, the Essentials plan is required. This plan includes 100 shipments per month. There are various paid-for tiers after that.
```
Comment (frenck) [inline source/_integrations/onetracker.markdown]:
```suggestion
The OneTracker integration allows one to track deliveries by [OneTracker](https://onetracker.app), a service that supports 490+ couriers worldwide. To use the tracking API functionality, the Essentials plan is required. This plan includes 100 shipments per month. There are various paid-for tiers after that.
```

File: source/_integrations/onetracker.markdown [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+The sensor value shows the number of packages that are not in `Delivered` state. As attributes are the number of packages per status.
+
+{% include integrations/config_flow.md %}
+
+{% configuration_basic %}
+email:
+  description: Email used for OneTracker account.
+  required: true
+  type: string
```
Comment (frenck) [inline source/_integrations/onetracker.markdown]: Configuration basic doesn't use these fields.

```suggestion
```

File: source/_integrations/onetracker.markdown [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+password:
+  description: Password used for OneTracker account.
+  required: true
+  type: string
```
Comment (frenck) [inline source/_integrations/onetracker.markdown]:
```suggestion
```

File: source/_integrations/onetracker.markdown [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+{% endconfiguration_basic %}
+
+## Setup
```
Comment (frenck) [inline source/_integrations/onetracker.markdown]:
```suggestion
## Prerequisites
```

File: source/_integrations/onetracker.markdown [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+To use this sensor, you need an [OneTracker Account](https://onetracker.app/signin). If you need to create an account you will need to download the [Android](https://play.google.com/store/apps/details?id=app.onetracker.android)
+or [iOS](https://itunes.apple.com/us/app/onetracker-package-tracking/id1409295535) application.
```
Comment (frenck) [inline source/_integrations/onetracker.markdown]: Considering this is needed before setting up this integration, I suggest on moving this chapter above the config flow section.

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Comment by packruler** [comment]
> ✅ Approved. Can be merged as soon as the parent PR gets merged. ⚠️ The `ha_release` might need an update before merging this one in.

Is there a proper placeholder version?

**Comment by frenck** [comment]
> Is there a proper placeholder version?

No. This is fine. We'll check when the upstream PR is ready.

**Comment by packruler** [comment]
https://github.com/home-assistant/core/pull/86921#issuecomment-1458583523

**Pull request closed without merging**